tests/test_network_groupnets.py

```diff
@@ -0,0 +1,34 @@
+import isi_sdk as isi_sdk
+from isi_sdk.rest import ApiException
+import urllib3
+
+urllib3.disable_warnings()
+
+import time
+
+from pprint import pprint
+
+# configure username and password
+
+isi_sdk.configuration.username = "root"
+isi_sdk.configuration.password = "a"
+isi_sdk.configuration.verify_ssl = False
+
+host = 'https://vnode2255.west.isilon.com:8080'
```

Comment:
Thanks for adding the additional test. Could it be modified to use `test_constants` rather than including an internal Isilon hostname? See [test_antivirus_policies.py](https://github.com/Isilon/isilon_sdk/blob/master/tests/test_antivirus_policies.py) for an example.